index.html

```diff
@@ -705,7 +705,9 @@ <h2 id="AudioContext">
           <dd>
             <p>
               This is a time in seconds which starts at zero when the context
-              is created and increases in real-time. All scheduled times are
+              is created and increases in real-time. At any given point, its value
+              represents the starting time of the next block of audio to be processed
```

Comment:
This is hard to implement in practice, because it depends on the event loop load, etc.